util/get_holes.py

```diff
@@ -0,0 +1,49 @@
+import json
+from pathlib import Path
+import requests
+from time import sleep
+
+base = 'https://xivapi.com/'
+spot = 'FishingSpot'
+columns = 'columns=ID,PlaceName.NameNoArticle,Item0.Singular,Item1.Singular,Item2.Singular,Item3.Singular,Item4.Singular,Item5.Singular,Item6.Singular,Item7.Singular,Item8.Singular,Item9.Singular'
+page = 1
+
+response = requests.get(f'{base}{spot}?{columns}&page={page}').json()
+# XIVAPI has a 1 request per second rate limit for requests without an API key
+sleep(1.5)
```

Comment:
I didn't actually try it without the sleeps; it might work just fine without them. It's just something that has to be run once per patch. If you're looking for a more robust and general interface for XIVAPI than just some requests, I could probably make it its own thing.

Reply:
I pulled out the sleep lines and it seemed to work fine; the rate limiting might not kick in until a certain amount of requests (there are only 3 for this)